applications/launchpad/gui-react/src/components/TBot/TBotPrompt/index.tsx

```diff
@@ -224,6 +227,10 @@ const TBotPrompt = ({
     return null
   }
 
+  const fadeOutGradient = floating
+    ? 'linear-gradient(to bottom, rgba(255, 255, 255, 1), rgba(255, 255, 255, 0.4))'
+    : 'linear-gradient(to bottom, rgba(250, 250, 250, 1), rgba(250, 250, 250, 0.4))'
+
```

Comment:
Because the `FadeOutSection` already depends on the `floating`, I think that we can move these gradients to `FadeOutSection` styles.